README.rst

```diff
@@ -34,7 +34,7 @@ Supported Python Versions
 Supported Django Versions
 -------------------------
 
--  1.7, 1.8, 1.9, 1.10, 1.11
+-  1.8, 1.9, 1.10, 1.11, 2.0
```

Comment:
I'll probably make some adjustments to the readme to have people use this version for Django 2 or higher and use the previous version for < 2.

Reply:
As you wish. 🤩